src/TuDa.CIMS.Shared/Entities/Chemical.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace TuDa.CIMS.Shared.Entities;
+
+/// <summary>
+/// An entity representing a chemical.
+/// </summary>
+public record Chemical()
```

Comment:
🔧 The inheritance to `AssetItem` is missing
```suggestion
public record Chemical : AssetItem
```